bdikit/schema_matching/best/matcher_factory.py

```diff
@@ -0,0 +1,63 @@
+import importlib
+from enum import Enum
+from typing import Mapping, Any
+from bdikit.schema_matching.best.base import BaseSchemaMatcher
+
+
+class SchemaMatchers(Enum):
+    SIMFLOOD = (
+        "similarity_flooding",
+        "bdikit.schema_matching.best.valentine.SimFloodSchemaMatcher",
```

Comment:
Is there any advantage of using strings here (+ dynamically instantiating it) instead of importing and using the actual class reference? I feel like importing the actual class could help catch bugs/typos earlier (e.g., through linting) and might help with code navigation if you're using an IDE that supports it, etc. I think I might be missing something here in relation to the advantages of this approach.

Reply:
@aecio, I noticed that simply importing bdikit used to take around 15 seconds because it was loading references to all primitives by default. Now, the import time is down to about 1 second.

The main issue with the previous implementation was that it loaded references to every primitive, even though not all would necessarily be used. For example, if a user is only performing schema matching, there’s no need to load the value-matching primitives. This issue could become more significant as more primitives are added.

Additionally, I'm planning to introduce optional primitives, which could cause the old implementation to raise errors when optional dependencies are missing.

I see more advantages in this implementation for these scenarios.